src/js/components/Image/image.stories.js

```diff
@@ -23,6 +23,16 @@ const Fit = () => (
   </Grommet>
 );
 
+const Fallback = () => (
+  <Grommet theme={grommet}>
+    <Image
+      fallback="//v2.grommet.io/assets/IMG_4245.jpg"
+      src="//v2.grommet.io/assets/IMG_4245_not_exists.jpg"
+    />
+  </Grommet>
+);
+
```

Comment:
this is great, can we also add a test to the new prop?

Reply:
i tried but can't get to hook on the onError event to actually test this. we would need to test this in a browser so that we get to see that the image loaded is the fallback one.

Reply:
Please see if we can take care of that on a separate PR.